.github/workflows/actions.yml

```diff
@@ -53,7 +53,7 @@ jobs:
       - name: Test applications with pytest
         if: ${{ steps.filter.outputs.applications == 'true' }}
         run: |
-          pytest keras/applications --cov=keras.applications
+          pytest keras/applications --cov=keras/applications
```

Comment:
Nice catch!

Reply:
I assume the issues on CI are simply because the tests are very large and the GH action is timing out. If it passes locally we're fine.

Reply:
> I assume the issues on CI are simply because the tests are very large and the GH action is timing out. If it passes locally we're fine.

The test failure with jax might be due to OOM (but reported as `Error: The operation was canceled.`):
https://github.com/actions/runner/issues/2468

As for torch, I can't reproduce the issue and identify the reason of the failure.

Reply:
Got it, I guess the applications tests had not been run since we made that config change for the torch CI. They're not run by default.